explorer/service/src/main/java/org/radixdlt/explorer/system/TestStateProvider.java

```diff
@@ -222,9 +222,9 @@ private boolean isMeasuring() {
         // TODO: Piers wants to transition to FINISHED state when >=90% of the data has been
         // validated (storedPerShard). Problem is that systemInfo is not normalised like in
         // MetricsProvide.calculateMetrics.      
-        if (currentState == TestState.STARTED) {
-            return (currentState.getStartTimestamp() + 60 * 60 * 1000) > System.currentTimeMillis();
-        }
+        //if (currentState == TestState.STARTED) {
+        //    return (currentState.getStartTimestamp() + 60 * 60 * 1000) > System.currentTimeMillis();
+        //}
```

Comment:
Just want to double check @blenessy: Is it okay, in your opinion, to remove this code? (If so, I'll go ahead and actually delete the lines as well).

Reply:
Yes I think so. Killing the network is a better (more dynamic) way of signalling test FINISHED to the Explorer.